Yoast/wordpress-seo
# Don't output breadcrumb schema on error pages

## Context


* We do not want breadcrumbs on error pages.

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where the breadcrumbs schema would be output on error pages.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure Breadcrumbs are enabled in Search Appearance - Yoast SEO.
* Browse to non-existing post.
* Check that you get 404 RC and grab schema. There should not be a `"@type":"BreadcrumbList"` graph included. See issue for what it was.

## UI changes

## Documentation

## Quality assurance


Fixes #14420
